fastlane/Fastfile

```diff
@@ -906,6 +928,34 @@ platform :ios do
   end
 end
 
+def extracted_release_notes_file_path(app:)
+  paths = {
+    wordpress: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Resources', 'release_notes.txt'),
+    jetpack: File.join(PROJECT_ROOT_FOLDER, 'WordPress', 'Jetpack', 'Resources', 'release_notes.txt')
+  }
+
+  # Good coding would have us handle the key not being found here. But, given
+  # this is a service method for an automation lane, we'd rather have the
+  # process break with a nil value and a human notice it immediately, than an
+  # incorrect file being used and the issue possibly going unnoticed for
+  # longer.
```

Comment:
By the same token, I didn't add a check on the input being a symbol.

What do you think of this approach? I wouldn't normally do it in a gem, but I see the lanes in a `Fastfile` as highly targeted and not meant to be re-used, so I think it's fine to trade safety for conciseness.